package.json

```diff
@@ -21,14 +21,14 @@
     "build:test": "NODE_ENV=test webpack --config webpack.config.js",
     "build:production": "NODE_ENV=production webpack --config webpack.config.js -p",
     "build:development": "NODE_ENV=development webpack -w --config webpack.config.js",
-    "generate-config": "generate-config -o 'dev-site-config' -s 'node_modules/terra-framework/packages/*/examples/*.site-page?(.jsx|.js)' -s 'node_modules/terra-core/packages/terra-site/src/examples/**/?(*Index|IconStatic|IconThemeable).jsx' -s 'node_modules/terra-clinical/packages/terra-clinical-site/src/examples/*/Index.jsx'",
     "heroku-postbuild": "npm install && npm run build:production",
     "lint": "npm run lint:js",
     "lint:js": "eslint --ext .js,.jsx . --fix",
     "start": "tt:serve",
     "start-heroku": "tt:serve-static --site build",
     "test": "npm run lint",
-    "deploy": "gh-pages -d build"
+    "deploy": "gh-pages -d build",
+    "tds:generate-app-config": "tds:generate-app-config"
```

Comment:
Do we need this script?

Reply:
no we don't, it was for testing purposes, i can remove